src/fluxVariability.m

```diff
@@ -137,12 +158,10 @@
 minFlux = zeros(length(rxnNameList), 1);
 
 if length(minNorm)> 1 || minNorm > 0
-    Vmin=zeros(nRxns,nRxns);
-    Vmax=zeros(nRxns,nRxns);
     %minimizing the Euclidean norm gets rid of the loops, so there
     %is no need for a second slower MILP approach
     allowLoops=1;
-else
+elseif OutputMatrix && isequal(method,'2-norm') && ~(length(minNorm)> 1 || minNorm > 0)

```

Comment:
Vmin and Vmax are generated when nargout>2 and nargin>7, the previous version had them computed even when they aren´t called.